python_modules/dagster/dagster/_core/definitions/result.py

```diff
@@ -31,6 +31,65 @@ class MaterializeResult(
     Attributes:
         asset_key (Optional[AssetKey]): Optional in @asset, required in @multi_asset to discern which asset this refers to.
         metadata (Optional[MetadataUserInput]): Metadata to record with the corresponding AssetMaterialization event.
+        check_results (Optional[Sequence[AssetCheckResult]]): Check results to record with the
+            corresponding AssetMaterialization event.
+        data_version (Optional[DataVersion]): The data version of the asset that was observed.
+    """
+
```

Comment:
`check_results` and `data_version` were missing from `MaterializeResult` docstring